spinners.json

```diff
@@ -835,5 +835,40 @@
 			"p",
 			"b"
 		]
+	},
+	"weather": {
+		"interval": 100,
+    "frames": [
+      "☀️ ",
+      "☀️ ",
+      "☀️ ",
+      "🌤 ",
+      "⛅️",
```

Comment:
You forgot a space here.

Reply:
That is on purpose - that emoji has a width of two spaces by itself.

Reply:
I can reproduce that on Terminal.app (macOS), but on iTerm.app they all have the same width. I think the Terminal.app behavior is incorrect. It makes no sense that just this character is different width from all the other ones.